### ISSUE TITLE
Support for the experimental syntax 'classProperties' isn't currently enabled

### ISSUE THREAD

**Issue by masmerino13** [ConnectyCube/connectycube-js-sdk-releases#47]
Hey guys,

Since I was experimenting an issue (https://github.com/ConnectyCube/connectycube-js-sdk-releases/issues/46) with the version `2.1.7` I decide to install the lastest version (actually 3.7.0) because seems the WS connection is lost after some time and I want to implement the ping strategy to preserve the connection alive.

The thing is that at the beginning I don't use the version 3+ because of this error, I did some changes in my side, but I haven't good luck fixing it.

The thing is, is this in your side?

Any suggestion is welcome.

This is my stack

```
React v16.13.1
Node v12.16.2
ConnectyCube v3.7.0
```

![image](https://user-images.githubusercontent.com/4874090/93010541-0eed3380-f54b-11ea-9cf3-3e390253e3d2.png)

**Comment by banshiAnton** [comment]
Hi @masmerino13 , you can find solution in this issue
https://github.com/ConnectyCube/connectycube-js-sdk-releases/issues/44

**Comment by masmerino13** [comment]
hi @banshiAnton, thanks a lot, that fix the problem!!

Now I was able to set the `ping` option in the config/init of the service, and seems it's working but bit sure if this is correct.

I'm using it like this:
```
const CONFIG = {
  chat: { ping: { enable: true, timeInterval: 60 } } // also I'm testing changing the value of the interval, also what the unit time for the interval?
};
ConnectyCube.init(CREDENTIALS, CONFIG);
```

But seems there are a lot of things happening, is this working as expected?

Seems everything is working well, but I'm not sure if it's okay that there are a lot of things happening in the websocket.

![ping](https://user-images.githubusercontent.com/4874090/93033644-57146080-f5f4-11ea-94a5-db811d486c20.gif)

Thanks for all your help guys!

**Comment by masmerino13** [comment]
after doing some test, seems if the `ping` feature is enable the chat doesn't works properly and the messages somethign aren't sent, calls and video calls doesn't works also ;)

**Comment by banshiAnton** [comment]
Hi @masmerino13 , our team find issue with ping
We will fix it in future releases
As fast solution you can just remove "ping" option from config https://developers.connectycube.com/js/messaging?id=ping

**Comment by banshiAnton** [comment]
We have released latest sdk version 3.6.3
https://www.npmjs.com/package/connectycube

**Comment by masmerino13** [comment]
Hi @banshiAnton , thank you, I have update to this version and it's working very well, thanks a lot!
